package-lock.json

```diff
@@ -9953,7 +9953,7 @@
             "integrity": "sha512-m62ShEObQ39CfralilEQRjH6oAMtNCV1xJyEx5LpRYUVN+EviphDgUc/F3hnYbADmkiNs67Y+3ylmlG7Lnu+FA==",
             "dev": true,
             "requires": {
-                "es5-ext": "0.10.53",
+                "es5-ext": "^0.10.50",
```

Comment:
Not sure what happened here, but I don't think these package-lock changes were intentional.

Reply:
Oi, nope. I just did an `npm i` because `gulp lint` was complaining locally. I'll fix.

Reply:
If you aren't already, I'd recommend enabling corepack, that way you run the version of npm declared in the package.json. Though, if a newer npm is doing this, we can totally bump that and relock.

Reply:
(Or the other way around, maybe we are relocking with a different version of npm; the GHA workflow doesn't enable corepack, something I forgot to do...)